learners/checkout.md

```diff
@@ -250,7 +250,7 @@ work for a checkout demo. Be sure to give yourself time to change course if your
 
 ### Sign-up and Set-up
 
-To sign up, select a session that works for you on [the Instructor Training Demonstration Sessions Etherpad]({{page.demopad}}), and add
```

Comment:
Sorry about that. This was one of the unique features of the IT curriculum that I did not catch.

This in particular was taking the `demopad` parameter from the YAML header and replacing it in this link. 

This was a case where a Jekyll solution was found where a more general (and portable) markdown solution existed.

This was the Jekyll solution:

```markdown
---
demopad: 'https://pad.carpentries.org/teaching-demos'
---

[Etherpad]({{ page.demopad }})
```

But it breaks in a non-Jekyll environment. Here's the general markdown solution:

```markdown
[Etherpad][demopad]

[demopad]: 'https://pad.carpentries.org/teaching-demos'
```